packages/odyssey/src/scss/odyssey.scss

```diff
@@ -49,6 +49,8 @@
 @import 'components/radio-button';
 @import 'components/radio-button-layout';
 @import 'components/select';
+@import 'components/status';
+@import 'components/status-layout';
```

Comment:
For my education, when do you break out styles in to `-layout`?

Reply:
This was a request from @rajaguruduraisamy-okta for web components - separate the layout styles when they need to be applied to / have context for any surrounding elements that are not available to the ShadowDOM.